projects/static/projects/css/style.css

```diff
@@ -46,24 +46,16 @@ html {
   font-family: "Trebuchet MS", sans-serif; }
 
 body {
-  background-color: #727272;
-  margin: 0;
-  padding: 0;
-  height: 100%;
-  width: 100%;
-  position: absolute; }
+  background-color: #FEFEFE !important; }
```

Comment:
#COVFEFE is a different color
![image](https://user-images.githubusercontent.com/6118864/29195840-f8d1226a-7e73-11e7-9125-e259a463cb4e.png)